README.md

```diff
@@ -1,94 +1,160 @@
-# ModSecurity Docker image
+ModSecurity Docker image
+========================
+
+[![dockeri.co](http://dockeri.co/image/vshn/modsecurity)](https://hub.docker.com/r/vshn/modsecurity/)
+
+[![Build Status](https://img.shields.io/docker/cloud/build/vshn/modsecurity.svg)](https://hub.docker.com/r/vshn/modsecurity/builds
+) [![GitHub issues](https://img.shields.io/github/issues-raw/vshn/modsecurity-docker.svg)](https://github.com/vshn/modsecurity-docker/issues
+) [![GitHub PRs](https://img.shields.io/github/issues-pr-raw/vshn/modsecurity-docker.svg)](https://github.com/vshn/modsecurity-docker/pulls
+) [![License](https://img.shields.io/github/license/vshn/modsecurity-docker.svg)](https://github.com/vshn/modsecurity-docker/blob/master/LICENSE)
 
 This image is based on the official [`owasp/modsecurity-crs`](https://hub.docker.com/r/owasp/modsecurity-crs) image.
 
 It contains the necessary tweaks to run on OpenShift.
 
-## Usage
-
-How to use the image
+Supported Tags
+--------------
+
+- [![latest](
```

Comment:
We should stay consistent with the symbol for lists through the whole document and therefore only use either `*` or `-`. I prefer `*`, because it is as close as possible to what it will actually be displayed as.